pkg/web_assets/templates/base.html

```diff
@@ -1,51 +1,39 @@
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <title>cheek</title>
-  <link rel="icon" type="image/x-icon" href="https://storage.googleapis.com/cheek-scheduler/cheek-64.png">
-  <link rel="stylesheet" href="/static/styles.css" />
-  <link rel="stylesheet" href="/static/tailwind.css" />
-  <script src="//unpkg.com/alpinejs" defer></script>
-  <script type="text/javascript" src="/static/script.js"></script>
-</head>
-
-<body class="bg-slate-800">
-  <div class="max-w-4xl mx-auto">
-    <div class="flex pt-6 items-end">
-      <a class="drop-shadow-md text-3xl font-extrabold text-center bg-gradient-to-r from-lime-200 to-blue-400 bg-clip-text text-transparent hover:from-red-400 hover:via-yellow-300 hover:via-green-500 hover:to-lime-200 hover:bg-gradient-to-l hover:animate-pulse"
-        href="/">cheek</a>
-      <div class="grow"></div>
-      <div class="col is-vertical-align is-right">
-        <a class="icon-ahref fill-lime-200 hover:fill-slate-200" href="https://github.com/datarootsio/cheek">
-          <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
-            <title>logo-github</title>
-            <g>
-              <path fill-rule="evenodd" clip-rule="evenodd"
-                d="M16,0.4c-8.8,0-16,7.2-16,16c0,7.1,4.6,13.1,10.9,15.2 c0.8,0.1,1.1-0.3,1.1-0.8c0-0.4,0-1.4,0-2.7c-4.5,1-5.4-2.1-5.4-2.1c-0.7-1.8-1.8-2.3-1.8-2.3c-1.5-1,0.1-1,0.1-1 c1.6,0.1,2.5,1.6,2.5,1.6c1.4,2.4,3.7,1.7,4.7,1.3c0.1-1,0.6-1.7,1-2.1c-3.6-0.4-7.3-1.8-7.3-7.9c0-1.7,0.6-3.2,1.6-4.3 c-0.2-0.4-0.7-2,0.2-4.2c0,0,1.3-0.4,4.4,1.6c1.3-0.4,2.6-0.5,4-0.5c1.4,0,2.7,0.2,4,0.5C23.1,6.6,24.4,7,24.4,7 c0.9,2.2,0.3,3.8,0.2,4.2c1,1.1,1.6,2.5,1.6,4.3c0,6.1-3.7,7.5-7.3,7.9c0.6,0.5,1.1,1.5,1.1,3c0,2.1,0,3.9,0,4.4 c0,0.4,0.3,0.9,1.1,0.8C27.4,29.5,32,23.5,32,16.4C32,7.6,24.8,0.4,16,0.4z">
-              </path>
-            </g>
-          </svg>
-        </a>
-
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>cheek</title>
+    <link rel="icon" type="image/x-icon" href="https://storage.googleapis.com/cheek-scheduler/cheek-64.png">
+    <link rel="stylesheet" href="/static/styles.css" />
+    <link rel="stylesheet" href="/static/tailwind.css" />
+    <script src="//unpkg.com/alpinejs" defer></script>
+    <script type="text/javascript" src="/static/script.js"></script>
+  </head>
+  <body class="bg-slate-800">
+    <div class="max-w-4xl mx-auto">
+      <div class="flex pt-6 items-end">
+        <a class="drop-shadow-md text-3xl font-extrabold text-center bg-gradient-to-r from-lime-200 to-blue-400 bg-clip-text text-transparent hover:from-red-400 hover:via-yellow-300 hover:via-green-500 hover:to-lime-200 hover:bg-gradient-to-l hover:animate-pulse" href="/">cheek</a>
+        <div class="grow"></div>
+        <div class="col is-vertical-align is-right">
+          <div id="version" class="text-gray-400 text-sm">Version 1.0.0</div>
```

Comment:
let's add the version where it says 
<img width="778" alt="image" src="https://github.com/user-attachments/assets/f96eee9e-cd65-4b80-b8fb-ba655559c673">

So we get something like `shows statuses up until the last 10 runs (running cheek v1.0.0)` (just inject the version in a `span` dedicated to it).

Reply:
<img width="988" alt="Screenshot 2024-10-08 at 15 57 52" src="https://github.com/user-attachments/assets/1da0ca26-6165-4600-85c2-2c86f5cc358c">